tests/rodeos_test.py

```diff
@@ -171,7 +171,8 @@ def get_info(self):
         loadSystemContract=False,
         specificExtraNodeosArgs={
             0: ("--plugin eosio::state_history_plugin --trace-history --chain-state-history --disable-replay-opts "
-                "--state-history-stride {0} " 
+          # stride unless it holds all the required data files -- it will cause test to fail 
+          #      "--state-history-stride {0} " 
```

Comment:
This should be removed instead of commented out. No need to have a `stateHistoryStride` that is not used.

Reply:
Agreed. Will take of it.